ui/server/src/main/scala/pl/touk/nussknacker/ui/definition/DefinitionPreparer.scala

```diff
@@ -29,8 +29,8 @@ object DefinitionPreparer {
   def prepareNodesToAdd(user: LoggedUser,
                         processDefinition: UIProcessDefinition,
                         isSubprocess: Boolean,
-                        nodesConfig: Map[String, SingleNodeConfig],
-                        componentsGroupMapping: Map[String, Option[String]],
+                        nodesConfig: Map[String, SingleComponentConfig],
```

Comment:
if you made renaming in types `node` -> `component` so maybe also change `nodesConfig` into `componentsConfig`?